nvdaHelper/remote/sysListView32.cpp

```diff
@@ -59,3 +59,46 @@ error_status_t nvdaInProcUtils_sysListView32_getColumnContent(handle_t bindingHa
 	*text = SysAllocString(lvItem.pszText);
 	return 0;
 }
+
+error_status_t nvdaInProcUtils_sysListView32_getColumnLocation(handle_t bindingHandle, const unsigned long windowHandle, int item, int subItem, RECT* location) {
+	RECT localRect {
+		// Returns the bounding rectangle of the entire item, including the icon and label.
+		LVIR_LABEL,  // left
+		// According to Microsoft, top should be the one-based index of the subitem.
+		// However, indexes coming from LVM_GETCOLUMNORDERARRAY are zero based.
+		subItem // top
+	};
+	if (!SendMessage((HWND)UlongToHandle(windowHandle), LVM_GETSUBITEMRECT, (WPARAM)item, (LPARAM)&localRect)) {
+		LOG_DEBUGWARNING(L"LVM_GETSUBITEMRECT failed");
+		return 1;
+	}
+	*location = localRect;
+	return 0;
+}
+
+error_status_t nvdaInProcUtils_sysListView32_getColumnHeader(handle_t bindingHandle, const unsigned long windowHandle, int subItem, BSTR* text) {
+	LVCOLUMN lvColumn = {0};
+	lvColumn.mask = LVCF_TEXT;
+	lvColumn.iSubItem = subItem;
+	lvColumn.cchTextMax = CBEMAXSTRLEN;
+	wchar_t textBuf[CBEMAXSTRLEN]{}; // Ensure that the array initialised with all zero values ('\0')
+	lvColumn.pszText= textBuf;
+	if (!SendMessage((HWND)UlongToHandle(windowHandle), LVM_GETCOLUMN, (WPARAM)subItem, (LPARAM)&lvColumn)) {
+		LOG_DEBUGWARNING(L"LVM_GETCOLUMN failed");
+		return 1;

```

Comment:
It would be great if these could be constants, I can't find any info on what these should be. If we internally use 1 for error and 0 for success, we should define these constants in a common header file with docs and use them directly.

Reply:
Honestly I believe this is out of scope for this pr. I used default Windows constants for now. You can find them here: 	// cchTextMax won't be changed to the actual number of characters in the buffer, so we can't use SysAllocStringLen here.
	// It would result in many null characters in the resulting BSTR.

If we really want to use constants all over the place, I think this should be done in a separate follow up, doing it once and for all.

Reply:
I think the paste wasn't what you expected here, would you mind double checking?

Reply:
Woops, I'm sorry. Corrected that.